viadot/sources/sap_rfc.py

```diff
@@ -132,9 +269,12 @@ def __init__(
         super().__init__(*args, credentials=credentials, **kwargs)
 
         self.sep = sep
+        self.replacement = replacement
         self.client_side_filters = None
         self.func = func
         self.rfc_total_col_width_character_limit = rfc_total_col_width_character_limit
+        # remove repeated reference columns
+        self.rfc_unique_id = list(np.unique(np.array(rfc_unique_id)))
```

Comment:
Is it required to use `numpy` here? There is an easy way to get unique values e.g. `list(set(rfc_unique_id))`

Reply:
implemented!